### DISCUSSION TITLE
Installazione ZBMini Extreme

### DISCUSSION THREAD

**Discussion by Tommaso6969** [indomus/forum#2390] [ZigBee, Z-Wave, Thread, Matter]
Ne ho già bruciato uno quindi mentre aspetto che me ne arrivi un altro vi chiedo aiuto.
allego gli schemi del ZBmini ed uno schizzo dei due interruttori deviatori che comandano un lampadario.
![Schema](https://user-images.githubusercontent.com/124528809/224476316-3e3cc075-c670-4e7b-b3ab-fae2a65faeb4.jpg)
<img width="620" alt="Zbminixt" src="https://user-images.githubusercontent.com/124528809/224476330-89e7ee21-b394-4a71-b858-85840e8be995.PNG">
Vi dico come l'ho rovinato: Lout sul 3a , Lin sul 3, s1 ed s2 su 1 e 2
Mi aiutate a non distruggere il prossimo?

**Comment by KC-inDomus** [comment]
> Ne ho già bruciato uno quindi mentre aspetto che me ne arrivi un altro vi chiedo aiuto. allego gli schemi del ZBmini ed uno schizzo dei due interruttori deviatori che comandano un lampadario. ![Schema](https://user-images.githubusercontent.com/124528809/224476316-3e3cc075-c670-4e7b-b3ab-fae2a65faeb4.jpg) <img alt="Zbminixt" width="620" src="https://user-images.githubusercontent.com/124528809/224476330-89e7ee21-b394-4a71-b858-85840e8be995.PNG"> Vi dico come l'ho rovinato: Lout sul 3a , Lin sul 3, s1 ed s2 su 1 e 2 Mi aiutate a non distruggere il prossimo?

chiedi a un elettricista...

**Comment by Tommaso6969** [comment]
Si, forse è meglio
